### PR TITLE
2024-03-06 Spell copy

### PR DESCRIPTION
**Pull request by 0xldr**
Spark proxy spell address is pending - labelled `$TBD` in text.

### CODE REVIEW

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Nice job, ldr.

File: governance/votes/Executive Vote - March 6, 2024.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
+---
+title: Template - [Executive Vote] Executive Vote - March 6, 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - March 6, 2024.md:2]: We typically include the items contained in the spell in the title for better searchability and consistency. Is there a reason for excluding them this time?

File: governance/votes/Executive Vote - March 6, 2024.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+summary: $summary
+date: 2024-03-06T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Executive Vote - March 6, 2024
```
Comment (VoteWizard) [inline governance/votes/Executive Vote - March 6, 2024.md:8]: Same issue with the title described above

**Review by VoteWizard — CHANGES_REQUESTED** [review]

File: governance/votes/Executive Vote - March 6, 2024.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Twelve Aligned Delegates will receive a total of 259.83 MKR as compensation for February 2024.
```
Comment (blimpa) [inline governance/votes/Executive Vote - March 6, 2024.md:19]: Are we no longer using "as detailed below"?

File: governance/votes/Executive Vote - March 6, 2024.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+* **Authorization**: [Atlas Section 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-operational-security)
+* **Proposal**: [Forum post](https://forum.makerdao.com/t/ad-derecognition-due-to-operational-security-breach-02-02-2024/23619/10)
+
+If this executive proposal passes the following transfer of a whistleblower bounty will take place:
+
+* Transfer **20.84 MKR** to @VeniceTree at [0xCDDd2A697d472d1e8a0B1B188646c756d097b058](https://etherscan.io/address/0xCDDd2A697d472d1e8a0B1B188646c756d097b058).
+
+**Note:** The MKR amount has been rounded to 2 decimal places to aid in the spellcrafting and verification process.
+
+### WBTC Vault [Target Available Debt `gap`](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) Changes
```
Comment (blimpa) [inline governance/votes/Executive Vote - March 6, 2024.md:92]: "(`gap`)"

File: governance/votes/Executive Vote - March 6, 2024.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+  * `max`: **28.5 million USDT**
+
+#### SparkLend Collateral Parameter Changes
+
+* **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmRU6mmi)
+* **Proposal**: [Forum post](https://forum.makerdao.com/t/feb-22-2024-proposed-changes-to-sparklend-for-upcoming-spell/23739)
+
+If this executive proposal passes the following collateral parameter changes will be made to SparkLend:
+
+**WETH**
```
Comment (blimpa) [inline governance/votes/Executive Vote - March 6, 2024.md:196]: (It's more semantically appropriate to make these headers, I think)

**Pull request merged**